.travis.yml

```diff
@@ -5,83 +5,79 @@ matrix:
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-comp7
-        - DOCKERTAG=latest
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-comp7
-        - DOCKERTAG=jnlp-slave
+        - DOCKERTAGSUFFIX=jnlp-slave
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-cos7-x86_64
-        - DOCKERTAG=latest
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-ppc64le
-        - DOCKERTAG=latest
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-aarch64
-        - DOCKERTAG=latest
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-cuda
-        - DOCKERTAG=9.2
+        - DOCKERTAGSUFFIX=9.2
         - CUDA_VER=9.2
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-cuda
-        - DOCKERTAG=10.0
+        - DOCKERTAGSUFFIX=10.0
         - CUDA_VER=10.0
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-cuda
-        - DOCKERTAG=10.1
+        - DOCKERTAGSUFFIX=10.1
         - CUDA_VER=10.1
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-cuda
-        - DOCKERTAG=10.2
+        - DOCKERTAGSUFFIX=10.2
         - CUDA_VER=10.2
 
     - os: linux
       env:
         - DOCKERIMAGE=miniforge3
-        - DOCKERTAG=latest
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-ppc64le-cuda
-        - DOCKERTAG=9.2
+        - DOCKERTAGSUFFIX=9.2
         - CUDA_VER=9.2
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-ppc64le-cuda
-        - DOCKERTAG=10.0
+        - DOCKERTAGSUFFIX=10.0
         - CUDA_VER=10.0
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-ppc64le-cuda
-        - DOCKERTAG=10.1
+        - DOCKERTAGSUFFIX=10.1
         - CUDA_VER=10.1
 
     - os: linux
       env:
         - DOCKERIMAGE=linux-anvil-ppc64le-cuda
-        - DOCKERTAG=10.2
+        - DOCKERTAGSUFFIX=10.2
```

Comment:
We already use `linux-anvil-ppc64le-cuda:10.2` in feedstocks. So this can't change.

Reply:
Ok, I’ll take another pass. I think I came up with a way to address this but also add the versioned images.